omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -128,9 +128,10 @@
     # Prepare the build env, these dependencies are only needed to build and
     # install the core integrations.
     #
+    command "#{pip} install hatchling==0.11.2"
     command "#{pip} install wheel==0.34.1"
     command "#{pip} install pip-tools==6.4.0"
-    uninstall_buildtime_deps = ['rtloader', 'click', 'first', 'pip-tools']
+    uninstall_buildtime_deps = ['rtloader', 'click', 'first', 'pip-tools', 'hatchling']
```

Comment:
If I read the code correctly, we currently uninstall the `uninstall_buildtime_deps` after installing `datadog_checks_base` and `datadog_checks_downloader`, but before we install the integration wheels. If we need `hatchling` for all integration wheels, should we move [that uninstall section](https://github.com/DataDog/datadog-agent/blob/ee69aa446143aa8e9440465dde5a31afca950fe7/omnibus/config/software/datadog-agent-integrations-py3.rb#L214-L221) to after the build of integrations? In that case I think it should be the last thing we run in [this block](https://github.com/DataDog/datadog-agent/blob/ee69aa446143aa8e9440465dde5a31afca950fe7/omnibus/config/software/datadog-agent-integrations-py3.rb#L302-L393).

Reply:
Will we need the `uninstall_buildtime_deps` that aren't `hatchling` for installing each integration?

Reply:
No

Reply:
Sure, makes sense